week-1/2-mandatory/4-tax.js

```diff
@@ -17,7 +24,15 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+/* salesPrice is the productPrice + taxValue*/
+/* taxValue is the productPrice * salesTax*/
+
+function formatCurrency(productPrice) {
+  let salesTax = 0.2;
+  let taxValue = productPrice * salesTax;
+  let salesPrice = productPrice + taxValue;
```

Comment:
Have you thought about how you could use your `calculateSalesTax` function inside this function to achieve the same result?  This is a good opportunity to practice call functions inside of functions!